### DISCUSSION TITLE
error 500 when viewing fork

### DISCUSSION THREAD

**Discussion by sl5net** [community/community#34667] [Other Feature Feedback, Questions, & Ideas]
when i go to my fork
https://github.com/sl5net/boonGUI
i got a error 500 page then as you could see below

![Screenshot_20220930_055945](https://user-images.githubusercontent.com/5634759/193187360-e7e8c5af-d4b6-408c-bdab-e7045edcbe56.png)

**Comment by andy5995** [comment]
500 sounds like an internal server error, but I don't see any problems reported yet when checking GitHub status. Hopefully it's only a temporary server glitch and will be resolved soon.

**Comment by radar-blip** [comment]
Working now. 👍🏻

**Comment by jfullstackdev** [comment]
it's fine on my end by now 

![image](https://user-images.githubusercontent.com/47092464/193276143-b2e5117c-2300-43d9-9a61-a8fec7f4f529.png)
